internal/dao/dp.go

```diff
@@ -123,6 +123,30 @@ func (d *Deployment) Restart(ctx context.Context, path string) error {
 	return err
 }
 
+// Pause a Deployment
+func (d *Deployment) Pause(ctx context.Context, path string) error {
```

Comment:
Given pause/resume are used in tandem, I think we could dry this up and use TogglePause(...) instead.
This will be inline with cronjobs and other resources where we offer toggling behavior
What do you think?